third-culture-software/bhima
# chore: release v1.12.0

**Changelog**

* deps: reformat release scripts and rm changelog (5a1bcffdc)
* Adding the date function on group by (81067ef11)
* fix(reports): improve Aux Cashbox Analysis (f72ac5138)
* Consider quantity > 0 (2d5d28cf2)
* fix(report); improve stock exit & inventory (eefbd2584)
* refactor(report): stock entry whitespace issues (1e77e091a)
* fix(reports): Month Consumption PDF fixes (9961b9b2f)
* fix link to installation (a628c8353)
* fix(stock): movement search flux displayValues (87f177996)
* fix(invoice): link patient via debtor_uuid (432634cc5)
* fix mergin conflicts (5fbe137df)
* fix the CMJ to CMM (6654d7216)
* fix(cash): bug fixes in search modal (6267dcb78)
* feat: improve release-it configuration (3ba171086)
* feat(employee registry) add a link to patient financial report (f9e4be474)
* re-enable skipped test (e8ef9d84a)
* fix order of lots results (de4146b24)
* fix(stock):  standardize requisition receipt (e27e2345a)
* fix(reports): fixes indent on operating account (3727a8fc4)
* fix(purchase): make DECIMALs compatible with PJ/GL (fcb7684e8)
* unknownWeekEnd to 123456789 (ce8eef38e)
* don't use GROUP as alias in MySQL query (f835e9995)
* fix some tests to send an integer id not a string (78cacf72a)
* make bhima MySQL8 compatible (cf2670047)
* test different versions of node (1ed69e0f1)
* fix: print colors in PDFs (764065beb)
* fix mergin conflict (985590821)
* Add enterprise option for changing CMM calculation (76863f4df)
* feat(stock requisition) add status filter (a8fa74d2d)
* consumption request (618d4ff1c)
* fix(stock): add ORDER BY to the stock movements q (c110c3ca8)
* improve(Report to compare invoiced items to received stock) - Clean migration/next.sql - Clean sql, delete not necessary condition - Remove not necessary extra line in table of report (5e4a68b8e)
* run CI with MariaDB (bff66dbc8)
* fix(Report to compare invoiced items to received stock) - Correction of error on the table - Display of the difference in costs (invoicing on distribution) instead of the difference in the cost of invoicing the item on compared to the cost of distribution - Correction in the joint concerning the unit cost of distribution (a4f3e053c)
* refactor(Report to compare invoiced items to received stock) - Set inventory items in invoice Alphabetically - set cell blank instead of put 0 - Don't display invoice has no consumable items - Set subtotal at the bottom of invoice (6bce38db4)
* Sanitaze : Code (9ab0d6efd)
* feature(Report to compaare invoiced items to received stock) - Implement the report to display what the patient has billed and what have received and what distribution without invoice Attibution (e006b2189)
* Resolve conflict with branch Master (634c872b5)
* feat(depot registry) add link to stock inventory (d49efc5a9)
* refactor(stock): move invoice reference on receipt (0988e24be)
* fix(reports): better empty message for fin. act. (27ee90deb)
* fix(Downloading employee registry as CSV and PDF crashes) - Modification of the request allowing to aggregate the filtered data - Use of the Uuid to the id of the screening of selected employees (74dab0beb)
* fix docs, currently only MySQL 5.7 is supported (3da6be40c)
* use db.convert (d284bb60a)
* test(multipayroll): refactor tests to use GridRow (cf43afa1e)
* feat(payroll): link employee from multiple payroll (8e8ecc8e4)
* Fix self requisition (74bab8baa)
* Add loading indicator (74b0a023e)
* document how to use docker to start MySQL server (d0138f36a)
* run installation tests in CI (1d26e7ced)
* fix e2e tests documentation (81470db08)
* update the migration file (b69589069)
* add dropdown-header (880bd3cc7)
* Add success message on the modal title (fa6bb8872)
* test: refactor fee center GridAction tests (9cdd8a00e)
* exit script when MySQL commands fail (d0b8e5950)
* obey $DB_HOST from .env file when creating DB (d5d85bc30)
* use of bhSubmit (57102a682)
* Fix text alignment in dropdown (27ae0f1af)
* Remove the not necessary column (ce4cc68c6)
* Fix loading indicators (57a5a9a5a)
* remove not necessary check (e8a970d0d)
* fix required MySQL version in documentation (fe2f1b325)
* feat(stock): link invoice from movements registry (4e698928e)
* feat(stock exit) show sold out inventory (0d03f1717)
* fix(cash): add clear() button to reference (d9bf15364)
* fix(purchases): use prev price in optimal PO (0c039635b)
* fix(stock report) correctly calculate sum when details are not displayed (48400dff5)
* fix sql linting issues (b70178af4)
* remove not necessary codes (09dabde0b)
* fix mergin conflicts (0aead71ae)
* docs(journal): add note about showing full txns (efb44da52)
* remove not necessary codes (4cc57026a)
* remove await in loop (744cc4d31)
* adjusting test to test database (db8c9a708)
* Adding period in stock adjustment (1bfb5ff73)
* fix and enhancements (55bd718a0)
* load lots by default in the grid (920e25675)
* fix mergin conflict (8a43c3881)
* fix and enhancements (833d04cde)
* load lots by default in the grid (6b5adaffc)
* Add some e2e tests (2a71889f4)
* Adding e2e tests (e4b575915)
* fix accounting for stock movement (ebfd84dc0)
* refactor inventory (e08cc22ee)
* reset inventory stock (ef75ea935)
* inventory adjustment refactoring (6c56f2bce)
